docs/4.1/admin-guide.md

```diff
@@ -303,9 +303,11 @@ auth_service:
     cluster_name: "main"
 
     authentication:
-        # default authentication type. possible values are 'local', 'oidc' and 'saml'
-        # only local authentication (Teleport's own user DB) is supported in the open
-        # source version
+        # default authentication type. possible values are 'local', 'oidc', 'saml' and 'false'
+        # 'false' is required for FedRAMP / FIPS, see 
+        #  https://gravitational.com/teleport/docs/enterprise/ssh_fips/#teleport-auth-server
+        #  only local authentication (Teleport's own user DB) & Github is supported in the open
```

Comment:
We say OSS supports Github but also don't specify that `github` is a possible value for for `authentication.type`, sounds kinda confusing

Reply:
Surprisingly `github` is the name of the authenticator . https://gravitational.com/teleport/docs/admin-guide/#github-oauth-20 

This must be from splitting OSS and Enterprise.

Reply:
Ow, I read this comment wrong. I've updated it.